.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           path: coverage/
 
   sonar-scan:
+    if: false
```

Comment:
what was the configuration problem with sonarcloud?

Reply:
It's not working with the monrepo setup because it is not being given the coverage correctly. I thought it was doing nothing at present; but at a second look it does still seem to be doing static analysis so we should probably